adapters/audienceNetwork/facebook.go

```diff
@@ -1,34 +1,38 @@
 package audienceNetwork
 
 import (
-	"bytes"
-	"context"
+	"crypto/hmac"
+	"crypto/sha256"
+	"encoding/hex"
 	"encoding/json"
 	"errors"
 	"fmt"
-	"io/ioutil"
-	"math/rand"
 	"net/http"
 	"strings"
 
+	"github.com/buger/jsonparser"
 	"github.com/golang/glog"
 	"github.com/mxmCherry/openrtb"
 	"github.com/prebid/prebid-server/adapters"
 	"github.com/prebid/prebid-server/errortypes"
-	"github.com/prebid/prebid-server/pbs"
-	"golang.org/x/net/context/ctxhttp"
+	"github.com/prebid/prebid-server/openrtb_ext"
 )
 
 type FacebookAdapter struct {
 	http         *adapters.HTTPAdapter
 	URI          string
 	nonSecureUri string
-	platformJSON json.RawMessage
+	platformId   string
```

Comment:
The Go style guide asks for `Id` to be `ID`. Same a few other structs in this PR.

Reply:
done